hack/community/readme.md

```diff
@@ -1,21 +1,49 @@
 # Cut a new community release
-The generate.sh script is called in the pre step to set up for the 
-aws-e2e-operator-test. The script copies over the WMCO community 
-bundle to the artifact directory while also replacing csv fields such as the 
-name, description, and date. Once generated into the artifact directory, the 
-bundle is used in the windows-e2e-operator test. After it passes, the bundle can
-be used to release a new community WMCO.
+CI contains a COMMUNITY environment variable that defaults to false, and is only
+set to true when a community-4.x branch PR is opened. When true, CI is 
+signaled to copy the repo's bundle into the test artifact directory, edit the 
+bundle, and then use the revised bundle in the PR's aws-e2e-operator test. After
+the test passes, the bundle can be copied from the PR's artifact directory and 
+used to release a new community WMCO.
```

Comment:
Would "bundle manifests", or "manifests", be more accurate here, instead of bundle?